server/storage/postgresql_test.go

```diff
@@ -0,0 +1,54 @@
+// +build postgresqldb
+
+// Initializes a PostgreSQL DB for testing purposes
+
+package storage
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	// Get the PostgreSQL connection string from an environment variable
+	dburl := os.Getenv("DBURL")
+	if dburl == "" {
+		logrus.Fatal("PostgreSQL environment variable not set")
+	}
+
+	for i := 0; i < 30; i++ {
+		gormDB, err := gorm.Open("postgres", dburl)
+		if err == nil {
+			err := gormDB.DB().Ping()
+			if err == nil {
+				break
+			}
+		}
+		if i == 29 {
+			logrus.Fatalf("Unable to connect to %s after 60 seconds", dburl)
+		}
+		time.Sleep(2 * time.Second)
```

Comment:
nit: actually, this line should be put before the `if`, otherwise when it prints out `after 60 seconds` is just only `58` seconds :)